### DISCUSSION TITLE
Need help.

### DISCUSSION THREAD

**Discussion by Boxter91** [jomjol/AI-on-the-edge-device#3112] [General]
### Issue

I cant get right readings from device.  Does anyone have idea why it is not working.
![Screenshot_20240613_200648_Chrome](https://github.com/jomjol/AI-on-the-edge-device/assets/98712552/847072dc-1fa4-4268-9507-5e7ea843f37d)

**Comment by SybexX** [comment]
try the rolling https://github.com/jomjol/AI-on-the-edge-device/actions/runs/9439572638
With the integrated zoom function you can enlarge the ROIs slightly and set them correctly.
And the reflection in the image could cause problems. https://github.com/jomjol/AI-on-the-edge-device/discussions/1949
